index.js

```diff
@@ -21,24 +21,9 @@ exports.handler = async (event, context) => {
       "x-ms-date": processingDate
     };
 
-    try {
-      let res = await request
+      let response = await request
```

Comment:
Any reason why not have it in one line with return statement ?
```suggestion
      return await request
```

Reply:
@mderazon it was done to resolve the `local system time not matching the time in Azure` issue